src/utils/misc.ts

```
@@ -2,7 +2,9 @@ import { database } from "./base"
 import { composeDashboard } from "./dashboard"
 import { getMainMessage } from "./embeds"
 
-import type { Guild, GuildPremiumTier, User, VoiceChannel } from "discord.js"
+import type { Guild, GuildPremiumTier, Snowflake, User, VoiceChannel } from "discord.js"
+import { getT } from "../generated/i18n"
```

ESLint

[error] Unable to resolve path to module '../generated/i18n'.

(import-x/no-unresolved)